frontend/src/components/VoteAdmin/index.scss

```
@@ -0,0 +1,85 @@
+@import '~/styles/variables';
+.wrap {
+  position: relative;
+  font-size: 12px;
+  color: BLACK_000;
+  text-align: center;
+  line-height: 2.5em;
+  border-radius: 4px;
+  background-color: $GRAY_009;
+  padding: 15px;
+  width: 90%;
+  height: 21%;
+  margin-top: 60px;
+}
+.wrapTitle {
+  display: flex;
+  justify-content: space-between;
+}
+.wrapVoteInfo {
+  display: flex;
+  flex-direction: column;
+}
+.wrapVoteInfo div {
+  display: flex;
+  padding-right: 5px;
+  margin-left: 3px;
+  align-items: center;
+}
+.target {
+  position: relative;
+  top: 1em;
+}
+.calender {
+  position: relative;
+  bottom: 1em;
+}
+.wrapButtons {
+  display: flex;
+  font-size: 13px;
+  flex-direction: row-reverse;
+  position: relative;
+  top: -10%;
+}
+.wrapButtons Button {
+  width: 56px;
+  height: 21px;
+  font-size: 11px;
+}
+.title {
+  font-size: 16px;
+}
+.voteState {
+  margin-right: 0;
+  margin-left: 6px;
+}
+.process {
+  color: $GREEN_003;
+}
+.end {
+  color: $GRAY_001;
+}
+.voteButton {
+  border-radius: 3px;
+  background-color: $BLUE_000;
+}
+.editButton {
+  margin-left: 5px;
+  border-radius: 3px;
+  background-color: $GREEN_004;
+}
+.closeButton {
+  margin-left: 5px;
+  border-radius: 3px;
+  background-color: $YELLOW_001;
+}
+.deleteButton {
+  margin-left: 5px;
+  border-radius: 3px;
+  background-color: $RED_002;
+}
+.disable {
+  margin-left: 5px;
+  border-radius: 3px;
+  background-color: $GRAY_002;
+}
```

이거는 클래스로 주는 방법도 있지만
button:disabled 선택자를 이용하는 방법이 있어서 이쪽을 사용하는게 더 깔끔합니다.
```suggestion
.button:disable {
  margin-left: 5px;
  border-radius: 3px;
  background-color: $GRAY_002;
}
```

그렇다면 이건 나중에 리팩토링 과정에서 베이직 컴포넌트 쪽으로 빼줄수도 있겠죠~